lib/commands/release.js

```diff
@@ -149,7 +149,7 @@ var ReleaseCommand = Command.extend({
     date: require('../strategies/date')
   },
   run: function(cliOptions) {
-    var SilentError = this.project.require('ember-cli/lib/errors/silent');
+    var SilentError = require('silent-error');
```

Comment:
Can you pull this out to the top of the file since it's no longer dependent on the project?